src/resources/js/directives.js

```diff
@@ -1661,20 +1661,28 @@ zaa.directive("zaaCheckbox", function () {
             };
             $timeout(function () {
                 $scope.init();
-            })
+            });
+
+            $scope.clicker = function() {
+                if ($scope.model == $scope.valueTrue) {
+                    $scope.model = $scope.valueFalse;
+                } else {
+                    $scope.model = $scope.valueTrue;
+                }
+            };
         }],
         template: function () {
             return '<div class="form-group form-side-by-side" ng-class="{\'input--hide-label\': i18n}">' +
                 '<div class="form-side form-side-label">' +
-                '<label for="{{id}}">{{label}}</label>' +
+                    '<label for="{{id}}">{{label}}</label>' +
                 '</div>' +
                 '<div class="form-side">' +
-                '<div class="form-check">' +
-                '<input id="{{id}}" ng-true-value="{{valueTrue}}" ng-false-value="{{valueFalse}}" ng-model="model" type="checkbox" class="form-check-input-standalone" ng-checked="model == valueTrue" />' +
-                '<label for="{{id}}"></label>' +
-                '</div>' +
+                    '<div class="form-check">' +
+                        '<input id="{{id}}" ng-true-value="{{valueTrue}}" ng-change="change()" ng-click="clicker()" ng-false-value="{{valueFalse}}" ng-model="model" type="checkbox" class="form-check-input-standalone" ng-checked="model == valueTrue" />' +
```

Comment:
You can add the following prop to make the input visible: `style="position: relative; visibility: visible; left: unset;"`